src/post/post.controller.ts

```diff
@@ -19,6 +19,13 @@ export class PostController {
     return foundPost;
   }
 
+  //search ALL post related with one user id
+  @Get('/searchPosts/all')
```

Comment:
`@Get('/all')`

because `get` means to search posts

Reply:
solved